lib/OpenQA/Controller/Worker.pm

```diff
@@ -44,11 +44,11 @@ sub worker_info($) {
     # puts job id in status, otherwise is idle
     if($job) {
         my $testdirname = $job->{'settings'}->{'NAME'};
-        my $results = test_result($testdirname);
-        my $modinfo = get_running_modinfo($results);
         $settings->{status} = "running";
         $settings->{jobid} = $job->{id};
-        $settings->{currentstep} = $modinfo->{running};
+        my $schema = Scheduler::schema();
```

Comment:
Here should be OpenQA::Scheduler::schema

I just noticed 'lib/OpenQA' is still put into @INC in compile check so that's probably why this passed that test